addon/components/flash-message.js

```diff
@@ -59,6 +60,14 @@ export default Component.extend({
     }
   }),
 
+  // Observer used here for backwards compatibility with Ember 1.11/1.12, replace
+  // with on('didRender') once possible.
+  _startTimers: on('didInsertElement', observer('flash', function() {
```

Comment:
In testing I found that the timers wouldn't start with just `didInsertElement` in some cases. I'm pretty sure this happens if you change the `flash` field on the component, for instance if the component renders the first message, it times out, and then it renders the second message without actually destroying the component.

Once 1.11 and 1.12 are dropped this can be solved with using `didRender`.